### PR TITLE
student profile

### PR DESCRIPTION
**Pull request by bc564**

### CODE REVIEW

File: people/bhagyasri-canumalla.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Bhagyasri Canumalla
+-------------
+
+![](photos/bhagya.jpg)
```
Comment (dblock) [inline people/bhagyasri-canumalla.md]: Please rename the JPG file to `bhagyasri-canumalla.jpg`, there might be other people with the same first name as you ;).

**Comment by dblock** [comment]
You also need to add a link to the README.md. Please make sure to _squash_ multiple commits, too.

**Comment by bc564** [comment]
All done!
ᐧ

Regards,
Bhagyasri Canumalla

**Comment by dblock** [comment]
There're two commits here. Please _squash_ them. This might be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

**Comment by dblock** [comment]
I still see two commits here, find me on slack if you need help squashing them.

**Pull request closed without merging**